src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+
+  margin: auto;
+  text-align: center;
+  max-width: 7 * ($cell-size + $gap-size);
+
+  &__day {
+    box-sizing: border-box;
+    width: $cell-size;
+    line-height: $cell-size;
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    transition-duration: 0.5s;
```

Comment:
you should also specify `transition-property` below or use a `transition` shorthand for both of these rules.